File: DKS-API/DTOs/SRfidMaintain.cs
using System;
using DKS_API.Helpers;

namespace DKS_API.DTOs
{
    public class SRfidMaintain : PaginationParams
    {
        public string recordTimeS { get; set; }
        public string recordTimeE { get; set; }
        public string area { get; set; }
    }
}

File: DKS-SPA/src/app/core/_models/s_rfid_maintain.ts
import { Pagination } from "./pagination";

export class SRfidMaintain extends Pagination {

  recordTimeS: string;
  recordTimeE: string;
  area: string;

  constructor() {

    super();
    this.isPaging = false; //不開分頁
    this.area = ''; //All

  }
  public setPagination(pagination: Pagination) {

    this.currentPage = pagination.currentPage;
    this.itemsPerPage = pagination.itemsPerPage;
    this.totalItems = pagination.totalItems;
    this.totalPages = pagination.totalPages;
  }
}
